chia/_tests/util/spend_sim.py

```diff
@@ -299,7 +309,7 @@ async def farm_block(self, puzzle_hash: bytes32 = bytes32(b"0" * 32)) -> tuple[l
         await self.new_peak(spent_coins_ids)
```

Comment:
it seems that you don't need `spent_coins_ids` anymore, now that you have `spent_coins`. I would think this could just be:
```suggestion
        await self.new_peak(spent_coins.keys())
```

Reply:
That won't work, `new_peak` expects `Optional[list[bytes32]]`.

Reply:
you mean it would need to expect `Optional[Sequence[bytes32]]` for it to work?